include/tdi/common/tdi_info.hpp

```diff
@@ -247,6 +247,17 @@ class TdiInfo {
    */
   const std::map<std::string, std::unique_ptr<tdi::Learn>> &learnMapGet() const;
 
+  /**
+   * @brief Get tables dependent on this table
+   *
+   * @param[in] id ID of the tdi::Table obj
+   * @param[out] pointer to vector of tdi_id
+   *
+   * @return status of the API call
+   */
+  tdi_status_t tablesThisTableDependsOnGet(
```

Comment:
Shouldn't this be just in the tdi_table_info.hpp?
Why do we want to support this here?

Reply:
tabelsThisTableDependsOnGet is just a thin wrapper that populates vector of 'dependee'.
The original c_frontend code called this wrapper. I maintained the call flow. 
I agree with you that it suffices to call just tableInfo API, dependsOnGet(). I will leave it to Weiqian how he would like to do this from c_frontend.

Reply:
Had a discussion with Weqian, looks like tdi_info level APIs are not really needed. So just providing a getter for depends_on_set.